genomic_data_service/region_indexer.py

```diff
@@ -10,11 +10,12 @@
 import time
 
 
-if "ES" in environ:
-    es_uri = [environ["ES"]]
-else:
+if "DOCKER" in environ:
     es_uri = ["elasticsearch"]
-es_port = 9200
+    es_port = 9200
+else:
+    es_uri = ["localhost"]
+    es_port = 9201
```

Comment:
Even though this might look simple, it is quite implicit and it is never clear from the caller's point of view what these settings are going to be. I think it would be appropriate to add a small argument parser and explicitly give the uri and port from the command line. The arguments should not contain any default value and should enforce being set.
This way it is abundantly clear from the Makefile which uri and which port are expected in each situation.